input/pagecontent/reference.md

```diff
@@ -1317,6 +1318,34 @@ In addition, authors should consult FHIR's [interpretation of ElementDefinition
       "A family member may not be biologically related due to adoption, blended families, etc."
   ```
 
+{%include tu-div.html%}
+The keyword `Characteristics` can be used to specify the type characteristics of the logical model being defined. These characteristics are represented on the logical model using the [Structure Type Characteristics extension](https://hl7.org/fhir/extensions/StructureDefinition-structuredefinition-type-characteristics.html) with a code value from the [TypeCharacteristicCodes value set](https://hl7.org/fhir/extensions/ValueSet-type-characteristics-code.html). Authors SHOULD use the `Characteristics` keyword instead of directly assigning this extension using assignment rules.
```

Comment:
> "Authors SHOULD use the `Characteristics` keyword

Is it even possible to set the value of an extension with a caret rule? I think we had to introduce the keyword because we had no other way to reach it.

Reply:
Yes, it's possible to assign characteristics using caret rules. The extension has to be repeated for each characteristic on the logical model. So, the following two definitions are equivalent:
```
Logical: LogicalWithKeyword
Characteristics: #has-units, #has-length

Logical: LogicalWithExtensions
* ^extension[0].url = "http://hl7.org/fhir/StructureDefinition/structuredefinition-type-characteristics"
* ^extension[0].valueCode = #has-units
* ^extension[1].url = "http://hl7.org/fhir/StructureDefinition/structuredefinition-type-characteristics"
* ^extension[1].valueCode = #has-length
```

Reply:
Right -- we decided to make it a keyword because it's fairly common, it's _required_ for any model you want to be able to point to in a reference, and fits within the type of metadata that usually goes in keywords.

Reply:
OK, thank you for the clarification. Looks ready to merge.